README.md

```diff
@@ -12,6 +12,8 @@ This collection has been tested against Juniper Networks Junos OS 18.4R1.
 
 This collection has been tested against following Ansible versions: **>=2.9.10**.
 
+For collections that support Ansible 2.9, please ensure you update your `network_os` to use the 
+fully qualified collection name (for example, `cisco.ios.ios`). 
```

Comment:
junos reference wold be better

Reply:
Should this be junipernetworks.junos.junos?